allofplos/article_class.py

```diff
@@ -676,47 +706,50 @@ def get_contributors_info(self):
                   .format(self.doi))
         return contrib_dict_list
 
-    def correct_or_retract_bool(self):
-        """Whether the JATS article type is a correction or retraction.
+    def amendment_bool(self):
```

Comment:
Why don't you call this `is_amendment` or `amendment` and make it a property? As long as self.type_ is cached as a string you shouldn't need to cache this.